coreneuron/mpi/lib/nrnmpi.cpp

```diff
@@ -52,110 +50,72 @@ void nrnmpi_init(int* pargc, char*** pargv) {
         nrn_assert(MPI_Init(pargc, pargv) == MPI_SUCCESS);
 #endif
     }
-    grp_bbs = MPI_GROUP_NULL;
-    grp_net = MPI_GROUP_NULL;
     nrn_assert(MPI_Comm_dup(MPI_COMM_WORLD, &nrnmpi_world_comm) == MPI_SUCCESS);
     nrn_assert(MPI_Comm_dup(nrnmpi_world_comm, &nrnmpi_comm) == MPI_SUCCESS);
-    nrn_assert(MPI_Comm_dup(nrnmpi_world_comm, &nrn_bbs_comm) == MPI_SUCCESS);
-    nrn_assert(MPI_Comm_rank(nrnmpi_world_comm, &nrnmpi_myid_world) == MPI_SUCCESS);
-    nrn_assert(MPI_Comm_size(nrnmpi_world_comm, &nrnmpi_numprocs_world) == MPI_SUCCESS);
-    nrnmpi_numprocs = nrnmpi_numprocs_bbs = nrnmpi_numprocs_world;
-    nrnmpi_myid = nrnmpi_myid_bbs = nrnmpi_myid_world;
+    nrn_assert(MPI_Comm_rank(nrnmpi_world_comm, &nrnmpi_myid_) == MPI_SUCCESS);
+    nrn_assert(MPI_Comm_size(nrnmpi_world_comm, &nrnmpi_numprocs_) == MPI_SUCCESS);
     nrnmpi_spike_initialize();
 
-    if (nrnmpi_myid == 0) {
+    if (nrnmpi_myid_ == 0) {
 #if defined(_OPENMP)
-        printf(" num_mpi=%d\n num_omp_thread=%d\n\n", nrnmpi_numprocs_world, omp_get_max_threads());
+        printf(" num_mpi=%d\n num_omp_thread=%d\n\n", nrnmpi_numprocs_, omp_get_max_threads());
 #else
-        printf(" num_mpi=%d\n\n", nrnmpi_numprocs_world);
+        printf(" num_mpi=%d\n\n", nrnmpi_numprocs_);
 #endif
     }
+
+    return std::make_tuple(nrnmpi_numprocs_, nrnmpi_myid_);
```

Comment:
```suggestion
    return {nrnmpi_numprocs_, nrnmpi_myid_};
```
should be fine even in C++14?